tardis/model/tests/test_csvy_model.py

```diff
@@ -34,24 +45,150 @@ def test_compare_models(filename):
+    input_isotopes_shape = input_isotopes.shape
+    assert model_isotopes_shape == input_isotopes_shape
+    npt.assert_array_almost_equal(
+        input_isotopes.to_numpy(),
+        csvy_model_to_test_abundances.raw_isotope_abundance.to_numpy(),
+    )
+
+
+def test_csvy_model_decay(csvy_model_to_test_abundances):
+    """Compare model abundance decay against and decay calculations 
+    done by hand.
+    For the calculations decay calulculations the following procedure is used:
+    Ni_halflife = 6.075 * u.d
+    Co_halflife = 77.233 * u.d
+
+    lambda_Ni = np.log(2) / Ni_halflife
+    lambda_Co = np.log(2) / Co_halflife
+
+    t = 4 * u.d means 4 days have passed since the explosion
+
+    def N1(N0, lambda1, t=4.0 * u.d):
+        return N0 * np.exp(-lambda1 * t)
+
+    
+    def N2(N1_0, lambda_1, lambda_2, t=4.0 * u.d):
+        return (lambda_1 * N1_0 
+                * (np.exp(-lambda_1 * t) / (lambda_2 - lambda_1)
+                + np.exp(-lambda_2 * t) / (lambda_1 - lambda_2)))
+
+     if the original Ni56 abundance for a given shell is 0.05, after 4 days:
+
+     cobalt_abundace_after_4_days = N2(0.05, lambda_Ni, lambda_Co)
+     nickel_abundace_after_4_days = N1(0.05, lambda_Ni)
+     iron_abundance_after_4_days = 0.05 - cobalt_abundace_after_4_days 
+                                    - nickel_abundace_after_4_days
+     In the hand_decayed_dataframe every row represents a specific element
+     and every column represent a shell"""
 
-    ref_abund = pd.DataFrame(np.array([[0.35,0.3, 0.6, 0.4],[0.65,0.7,0.4,0.6]]))
-    ref_abund.index.name = 'atomic_number'
-    ref_abund.index = np.array([1, 2])
+    decay_index = pd.Index([1, 2, 26, 27, 28], name="atomic_number")
+    hand_decayed_abundance = pd.DataFrame(
+        [
+            [0.0, 0.33, 0.3, 0.5, 0.4, 0.2],
+            [0.98, 0.64, 0.6, 0.4, 0.55, 0.79],
+            [
+                0.00013977843354947162,
+                0.00020966765032420787,
+                0.0006988921677473642,
+                0.0006988921677473642,
+                0.0003494460838736821,
+                6.988921677473581e-05,
+            ],
+            [
+                0.007188928223953217,
+                0.010783392335929825,
+                0.035944641119766085,
+                0.035944641119766085,
+                0.017972320559883043,
+                0.0035944641119766084,
+            ],
+            [
+                0.012671293342497312,
+                0.019006940013745966,
+                0.06335646671248656,
+                0.06335646671248656,
+                0.03167823335624328,
+                0.006335646671248656,
+            ],
+        ],
+        index=decay_index,
+    )
 
-    assert csvy_abund.equals(ref_abund)
\ No newline at end of file
+    model_decayed_abundance_shape = csvy_model_to_test_abundances.abundance.shape
+    hand_decayed_abundance_shape = hand_decayed_abundance.shape
```

Comment:
you didn't decay it by hand 😉 - maybe reference_abundance_shape or so - is better. @harpolea what do you suggest ?

Reply:
yes I agree, calling it a 'reference' abundance rather than a 'hand' abundance makes more sense